components/textbox/textbox_links.jsx

```diff
@@ -143,3 +146,9 @@ export default class TextboxLinks extends React.Component {
         );
     }
 }
+
+const mapStateToProps = (state) => ({
```

Comment:
Instead of adding this in the same file, it should be moved out into its own file